content/components/data-table.mdx

```diff
@@ -600,6 +600,20 @@ The table must have a title, and can optionally have a subtitle to describe more
   src="https://user-images.githubusercontent.com/2313998/205361249-763772e1-d856-479e-a632-d185c27095db.png"
 />
 </Box>
+
+### Pagination buttons
+
+<Box as="p" flex={1}>
+  The pagination control for a DataTable uses `button` elements, and not `a` elements. This is because activating the 
+  pagination control requests data, and does not have a corresponding URL to link to.
```

Comment:
We shouldn't have line breaks in `.mdx` text blocks unless they're used to add a breaking space or create a new paragraph.

Reply:
Note: this only applies if we're not wrapping the text in a `<p>` or `<Box as="p">`